### PR TITLE
Apr 1444 criminal compliance org composition creditors

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
+﻿using FluentAssertions;
+using MediatR;
```
Comment (MarkFCain) [inline src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs]: this shouldn't be here

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+﻿using MediatR;
```
Comment (MarkFCain) [inline src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs]: this shouldnt be here

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            _applyApiClient = applyApiClient;
+            _contextAccessor = contextAccessor;
+            _gatewayValidator = gatewayValidator;
+            _orchestrator = orchestrator;
+            _logger = logger;
+        }
+
+
+        [HttpGet("/Roatp/Gateway/{applicationId}/Page/5-10")]
```
Comment (MarkFCain) [inline src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs]: this 5-10 code will be going, you'll need to wait for the overview changes to go in.  I expect it to be replaces with 'composition-with-creditors' or similar

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        public async Task<IActionResult> GetOrganisationCompositionCreditorsPage(Guid applicationId)
+        {
+            var username = _contextAccessor.HttpContext.User.UserDisplayName();
+            var viewModel = await _orchestrator.GetCriminalComplianceCheckViewModel(new GetCriminalComplianceCheckRequest(applicationId, GatewayPageIds.CCOrganisationCompositionCreditors, username));
+            return View("~/Views/Roatp/Apply/Gateway/pages/OrganisationCriminalComplianceChecks.cshtml", viewModel);
+        }
+
+        [HttpPost("/Roatp/Gateway/{applicationId}/Page/5-10")]
```
Comment (MarkFCain) [inline src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs]: see above comment about 5-10

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml [hunk 5]
```diff
@@ -0,0 +1,298 @@
+﻿@using SFA.DAS.AssessorService.ApplyTypes.Roatp
+@using SFA.DAS.AssessorService.Domain.Extensions
+@model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.OrganisationCriminalCompliancePageViewModel
+
+@{
+    var heading = "Legal name check";
+    var caption = "Organisation checks";
```
Comment (MarkFCain) [inline src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml]: This caption needs updating

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml [hunk 6]
```diff
@@ -0,0 +1,298 @@
+﻿@using SFA.DAS.AssessorService.ApplyTypes.Roatp
+@using SFA.DAS.AssessorService.Domain.Extensions
+@model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.OrganisationCriminalCompliancePageViewModel
+
+@{
+    var heading = "Legal name check";
```
Comment (MarkFCain) [inline src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml]: this heading needs updating

Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml]: Updated view

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs (hunk 4 above)
Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs]: Replaced with url tags

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs (hunk 3 above)
Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs]: Replaced with url tags

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs (hunk 2 above)
Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs]: Removed unused usings

File: src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs (hunk 1 above)
Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs]: Removed unused usings

**Pull request closed without merging**